### PR TITLE
HOTFIX address node vulns

### PR DESCRIPTION
**Pull request by dasormeter**

### CODE REVIEW

**Comment by stefanobaghino-da** [comment]
It looks like you have been running some commands on a stale `node_modules` installation that probably pointed to a warning, which I'm not sure points to some kind of vulnerability.

If you could possibly add some details on why this would address vulnerabilities, that would be very helpful.

Also please note that you should use the following command to add a `devDependency`:

    npm i -DE <dependency_name>

This will make sure to keep the package manifest and lock file synchronized for reproducible builds. Failing to do so will cause the CI to reject your contribution (which is what you can observe in this case).

##### Regarding vulnerabilities

Regarding vulnerabilities in the project's dependencies, we currently have 3 moderate vulnerabilities on the current `master`. Unfortunately it's quite difficult to deal with them:
- two of them are on `grpc_tools_node_protoc_ts` (which generates the TypeScript type definitions used to interact with the code generated by gRPC -- and would be quite a big deal to replace), both due to [this Node.js security advisory](https://nodesecurity.io/advisories/745). Please note that `grpc_tools_node_protoc_ts` is a development tool, so that vulnerability surface area applies to the company's laptops used for development and CI resources. Given the advisory, it would seem to me as a mild risk, but probably you have a better insight into this. **Please note that upgrading to the latest version actually makes the matter worse as it introduces a vulnerability marked as `high`.**
- one is due to `typedoc` (that is used to generate reference documentation from docstrings in the code and could probably be even a bigger deal to replace), in particular with regards to [this Node.js security advisory](https://nodesecurity.io/advisories/812). Here the security surface area seem to be smaller, as this tool is exclusively used to manually generate reference documentation on release and this process is currently not automated on CI. Unfortunately here the reported vulnerability is even more difficult to fix because the project is already using the latest version of TypeDoc, so the best possible way to fix this would probably be to contribute upstream to the dependency.

To recap:
1. please let me know more details about this PR (I would suggest running `rm -rf node_modules` to start from a fresh situation and possibly close if the warning is gone)
2. it may be worth having a piece of your mind on [existing vulnerabilities reported by `npm audit`](#regarding-vulnerabilities) -- but of course feel free to disregard it if you don't feel like there's a problem at all

**Comment by dasormeter** [comment]
okay this one is a bit old so i'll rerun again to see what it was that flagged this lib

**Comment by dasormeter** [comment]
here's what i get for vulnerabilities

`brianhealey:daml-js/ (master✗) $ npm audit                                                                                                                                                           [15:32:50]

                       === npm audit security report ===

# Run  npm install --save-dev grpc_tools_node_protoc_ts@2.5.1  to resolve 2 vulnerabilities
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ underscore.string                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ grpc_tools_node_protoc_ts [dev]                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ grpc_tools_node_protoc_ts > handlebars-helpers >             │
│               │ helper-markdown > remarkable > argparse > underscore.string  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/745                       │
└───────────────┴──────────────────────────────────────────────────────────────┘


┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ underscore.string                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ grpc_tools_node_protoc_ts [dev]                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ grpc_tools_node_protoc_ts > handlebars-helpers > helper-md > │
│               │ remarkable > argparse > underscore.string                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/745                       │
└───────────────┴──────────────────────────────────────────────────────────────┘


┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ marked                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.6.2                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ typedoc [dev]                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ typedoc > marked                                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/812                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 3 moderate severity vulnerabilities in 1894 scanned packages
  run `npm audit fix` to fix 2 of them.
  1 vulnerability requires manual review. See the full report for details.`
